### DISCUSSION TITLE
UI : Increase size of Treatment entry

### DISCUSSION THREAD

**Discussion by spanosp** [NightscoutFoundation/xDrip#3240] [Ideas]
If possible to have a larger keyboard for treatment entry, I beleieve this will be useful for people with 'fat fingers' and easier entry using one hand only or for entry on the go.

Similarly the Treatment icon can be larger and further away from the Treatment Note icon.

TY

**Comment by Navid200** [comment]
Thanks for the suggestion.

Some of the layouts are chosen so that if the device is very small, it would still look reasonable on the screen.
I am not sure if we can use a keyboard that can become larger depending on the number of pixels on the screen.

**Reply by spanosp** [reply]
Could it be an option to increase size by 50%?

**Reply by Navid200** [reply]
Do you mean the number keys?  Or, do you mean the 5 buttons at the bottom?

**Reply by spanosp** [reply]
Everthing really, the number keys mostly as used more frequently

**Reply by Navid200** [reply]
I just tried "Add calibration".
After the keypad came up, I rotated my phone to landscape.  This changed the keypad such that I could adjust the size and extend it to make it larger.

Let me see if we can add the same capability to the treatment menu.

**Reply by spanosp** [reply]
..and maybe seperate the Treatment icon and Treatment Note icon slightly more
